Remove setting the EDITOR and PAGER environment variables in /etc/profile,
so /etc/profile.d scripts can properly override them by only doing so if the
user hasn't already set them to something else.

--- a/etc/profile
+++ b/etc/profile
@@ -10,11 +10,6 @@
 	. /etc/profile.env
 fi

-# You should override these in your ~/.bashrc (or equivalent) for per-user
-# settings.  For system defaults, you can add a new file in /etc/profile.d/.
-export EDITOR=${EDITOR:-/bin/nano}
-export PAGER=${PAGER:-/usr/bin/less}
-
 # 077 would be more secure, but 022 is generally quite realistic
 umask 022


